api/resources_portal/test/views/test_organization_invitation.py

```diff
@@ -213,3 +213,27 @@ def test_put_request_without_authentication_fails(self):
 
         response = self.client.put(self.url, invitation_json)
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_delete_request_deletes_invitation(self):
+        self.client.force_authenticate(user=self.invitation.requester)
+        invitation_id = self.invitation.id
+        response = self.client.delete(self.url)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(OrganizationInvitation.objects.filter(id=invitation_id).count(), 0)
+
+    def test_delete_request_from_non_requester_fails(self):
+        self.client.force_authenticate(user=self.invitation.request_reciever)
+        response = self.client.delete(self.url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_delete_request_from_unauthenticated_fails(self):
+        self.client.force_authenticate(user=None)
+        response = self.client.delete(self.url)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_delete_request_only_soft_deletes_objects(self):
+        self.client.force_authenticate(user=self.invitation.requester)
+        invitation_id = self.invitation.id
+        response = self.client.delete(self.url)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(OrganizationInvitation.deleted_objects.filter(id=invitation_id).count(), 1)
```

Comment:
This seems identical to `test_delete_request_deletes_invitation` except with a different final assertion. I think they could just be combined to a single test with both assertions. (This is actually basically what I was requesting with the subsequent GET requests.)